src/util/Utils.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef UTIL_UTILS_H_
+#define UTIL_UTILS_H_
+
+#include <iterator>
+#include <string>
+#include <vector>
+
+#include <folly/String.h>
+
+namespace nebula {
+namespace util {
+
+template <typename Container, typename Fn>
+std::string join(const Container& container, Fn fn, const std::string& delimiter = ",") {
```

Comment:
Could add a template specialization for vector itself to avoid extra copy.

Reply:
I want to let the `join` function adapt to sequential container type, like `std::vector` or `std::list`. And the `container` parameter is a const reference which will not copy the real parameter value.